### PR TITLE
New "keys" option to select which keys you want to import

### PR DESCRIPTION
**Pull request by felixhorro**
If you only want some of the keys from the object imported, you can supply the keys you want the plugin to load.

### CODE REVIEW

**Comment by ajwhite** [comment]
Support added to existing `options.environment` in #61

**Comment by ajwhite** [comment]
This is now published under the [`v1.4.0` release](https://github.com/ajwhite/gulp-ng-config/releases/tag/v1.4.0)

**Comment by felixhorro** [comment]
Thanks, but #61 doesn't fit my needs.

As an example, I want to expose app version as an angular constant, so I use gulp-ng-config to get it from the package.json of my project. I only need "version" property, don't want to make "name", "description", "dependencies", etc. constants.

Thanks!

**Comment by ajwhite** [comment]
I see, so you want the specified key to actually be added to the constant, _not just the children_.

For example:
```json
{
  "wanted": "foobar",
  "environmentA": {
    "hello": "world"
  }
}
```

Using `environment`
```js
gulpNgConfig('appConfig', {
  environment: 'environmentA'
})
```
Will produce
```js
angular.module('appConfig', [])
.constant('hello', 'world')
```

----

What you want is to actually specify that key and have it become part of the output:
```js
gulpNgConfig('appConfig', {
  keys: 'environmentA'
})
```
Producing
```js
angular.module('appConfig', [])
.constant('environmentA', {'hello': 'world'})
```

**Comment by ajwhite** [comment]
@felixhorro let me know on the above, we can still look at this

**Comment by felixhorro** [comment]
@ajwhite You're right, I need the key to be imported, not the children.

Conflicts solved!

File: test/stream.js [hunk 1]
```diff
@@ -479,6 +479,27 @@ describe('gulp-ng-config', function () {
           });
       });
     });
+    describe('keys', function () {
+      it ('should select some keys if a keys option is supplied', function (done) {
+        var expectedOutput = fs.readFileSync(path.normalize(path.join(__dirname, 'mocks/output_22.js')));
```
Comment (ajwhite) [inline test/stream.js]: Could you add this file with the expected output?

Comment (felixhorro) [inline test/stream.js]: Ok, file added!
Thanks!

**Comment by felixhorro** [comment]
:tada: :tada: :confetti_ball: :confetti_ball: :tada: :tada: 
Thanks!

**Pull request merged**